docker-compose-arm.yml

```diff
@@ -0,0 +1,21 @@
+# Docker compose file to manage your deployed images.
+# Use MySQL server 5.7 and latest Scada-LTS local build.
+# Using attached webapps folder as developer you will be able to modify the static content from host os.
+# Attach shell to stop the tomcat instance and then you will be able to run in JPDA mode.
+version: '3'
+services: 
+    database:
+        container_name: mysql
+        image: biarms/mysql:5.7
```

Comment:
1. Is this file necessary? Docker itself will select the image of the appropriate architecture, based on the host architecture;
2. We use MySQL server version +8;
3. We should use official images from https://hub.docker.com/r/mysql/mysql-server/tags
and here linux/arm64 is an alias for linux/arm64/v8;

Reply:
I get this error when using mysql:8.0 at startup

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/30082001/2697ed9d-aecf-4c6b-a0fd-1342a7363ac3)
[scadalts-docker-logs.txt](https://github.com/user-attachments/files/15532351/scadalts-docker-logs.txt)

official mysql 5.7 image does not have an arm64 version

Reply:
1. We no longer support mySQL 5.7. Use min version 8.0.32. Is corrected docker-compose file for MySQL 8: https://github.com/SCADA-LTS/Scada-LTS/blob/develop/docker-compose.yml;

Reply:
Perfect, it works with that mysql version. I tested this build with a Raspberry Pi 4B 4GB